packages/roosterjs-content-model-types/lib/event/LogicalRootChangedEvent.ts

```diff
@@ -0,0 +1,6 @@
+import type { BasePluginEvent } from './BasePluginEvent';
+
+/**
+ * Fired when the logical root changes
+ */
+export interface LogicalRootChangedEvent extends BasePluginEvent<'logicalRootChanged'> {}
```

Comment:
Better pass the new logicalRoot with the event object so plugin knows where it has been changed to.